docs/platform/manager/accounts-and-passwords-new-manager/index.md

```diff
@@ -0,0 +1,162 @@
+
+## Users and Permissions
+
+You can grant other people access to your Linode Manager account by creating *users* and assigning *permissions* to restrict access to certain areas of the control panel. Adding users and configuring permissions is useful for groups that need to grant all team members access to the Linode Manager, or organizations that just want their billing department to have a separate account to receive invoices and billing information.
+
+ {{< note >}}
+A single user was automatically created for your account when you signed up for Linode. If you will be the only person accessing the Linode Manager, you won't need to create any other users on your account.
+{{< /note >}}
+
+### Adding a User
+
+When you add a user, you give that individual permission to log in to the Linode Manager and, depending on the access level they are assigned, receive email notifications. There are two types of users: *Restricted users*, who have limited access to the Linode Manager, and *unrestricted users*, who have full access.
+
+Here's how to add a user to your Linode account:
+
+1.  Log in to the [Linode Manager](https://cloud.linode.com).
+2.  Click the **Account** menu on the left-hand side.
+3.  Click the **Users** link.
+    [![Click on the 'Account' link and then click 'Users'](accounts-overview-small.png "Click on the 'Account' link then click 'Users')](accounts-overview.png)
+4.  Select the **Add a User** link. The *Add a User* menu appears.
+    ![The 'Add a User' menu](accounts-add-a-user-menu.png "The 'Add a User' menu.")
+5.  Enter a username for the user in the **Username** field.
+6.  Enter the user's email address in the **Email** field. A password creation link will be sent to the new user's email address.
+7.  To give the new user restricted access, toggle the **Restricted Access** button.
+8.  Click **Submit**.
+
+If you granted the user full access, the account will be created and no further action is required. If you opted to restrict the user, follow the instructions in the next section.
+
+### Setting Permissions
+
+Setting permissions restricts a user's access to certain areas of the Linode Manager. For example, you could limit a user to a single Linode and prevent them from removing the Linode or adding extra services. Don't worry--these settings aren't permanent. You can add or remove access for a user at any time in the future.
+
+Here's how to set a user's access permissions:
+
+1.  Click the **Account** link on the left-hand side of the page.
+2.  Click the **Users** link.
+3.  Click on the user in the list and select the **User Permissions** tab. The webpage shown below appears.
```

Comment:
This is a single-page app, it's all the same webpage. This could probably be called a form or a panel.